lib/pages/explore/menu_result_card.dart

```diff
@@ -14,24 +15,31 @@ Card createMenuSearchResultCard(MenuSearchResult menuResult, bool selected) {
           semanticLabel: 'Text to announce in accessibility modes',
         ),
         SizedBox(width: 2),
-        Text("Menu",
-            style: TextStyle(
-              fontSize: 12,
-            )),
+        Text("Menu", style: Theme.of(context).accentTextTheme.headline4)
       ],
     ),
     Text(menuResult.name, style: TextStyle(fontSize: 16)),
-    Row(
-      children: [
-        Text("Mon", style: TextStyle(fontSize: 12)),
-        SizedBox(width: 5),
-        Text("Tues", style: TextStyle(fontSize: 12)),
-        SizedBox(width: 5),
-        Text("Wed", style: TextStyle(fontSize: 12))
-      ],
-    )
+    Row(children: getDays(menuResult.days, context))
   ];
   //TODO: LOAD BACKGROUND DYNAMICLY
   return createSearchResultCard(widgets,
       background: "assets/images/y_6cfb1008.jpg", selected: selected);
 }
+
+List<Widget> getDays(List<int> days, BuildContext context) {
+  List<Widget> dayTextWidgets = List();
+  String dayString = "";
+  days.forEach((day) {
+    if (day == 1) dayString = "Mon";
+    if (day == 2) dayString = "Tue";
+    if (day == 3) dayString = "Wed";
+    if (day == 4) dayString = "Tue";
+    if (day == 5) dayString = "Fri";
+    if (day == 6) dayString = "Sat";
+    if (day == 7) dayString = "Sun";
```

Comment:
```suggestion
    switch(day) {
      case 1: dayString = "Mon"; break;
      case 2: dayString = "Tue"; break;
      case 3: dayString = "Wed"; break;
      case 4: dayString = "Tue"; break;
      case 5: dayString = "Fri"; break;
      case 6: dayString = "Sat"; break;
      case 7: dayString = "Sun"; break;
    }

```
veldig optional mere fluffy men mere efektiv siden man kun velge den dagen sin index man bruke og ikke teste på alle hver gang. veldig mikroptimalisering men spare sikkert minst et par nano sec

Reply:
xD helt enig, haha